mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -577,6 +577,17 @@ def _get_v3io_frames_client(
             token=v3io_access_key,
         )
 
+    @staticmethod
+    def _get_endpoint_filter(endpoint_id: Union[str, list[str]]):
+        if isinstance(endpoint_id, str):
+            return f"endpoint_id=='{endpoint_id}'"
+        elif isinstance(endpoint_id, list):
+            return f"endpoint_id IN({str(endpoint_id)[1:-1]}) "
```

Comment:
With v3io with have a problem to run a query which is more when 4096 bytes... please check this or just limit the list to reasonable len (±50) and if it more when that please raise explainable error.